v63002/inref/gtm8165.m

```diff
@@ -0,0 +1,20 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+tpnotacid
+	tstart ():(serial:transaction="BA")
+	if $trestart>2 write /wait(.999)
```

Comment:
Also need to test that critical section is held before and not held after the write /wait. Can be checked by $$^%PEEKBYNAME("node_local.in_crit","DEFAULT"). Mention the test of that in the reference file too.